### PR TITLE
sieve: add new exercise

### PR DESCRIPTION
**Pull request by filiptoskovic**

### CODE REVIEW

File: exercises/sieve/uSieve.pas [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  procedure Output;
+  var
+    i : integer;
+    LOut : TList<integer>;
+  begin
+    LOut := TList<integer>.Create;
+    for i :=  2 to AInp do
+      if Sieve[i - 2] then
+        LOut.Add(i );
```
Comment (rpottsoh) [inline exercises/sieve/uSieve.pas]: Minor nit pick.  Remove the space after `i`.

**Comment by rpottsoh** [comment]
@filiptoskovic you will also need to rename `uSieve.pas` to `uSieveExample.pas`.

**Comment by rpottsoh** [comment]
Thanks for your hard work @filiptoskovic.  I am going to go ahead and merge this.  Have you earned your TShirt yet? :wink:

**Comment by filiptoskovic** [comment]
Yes, thank you on info. This is great site, yesterday one of my friend, long time Delphi programmer, asked me how I have improved that much in few weeks (actually almost two months).  Exercism is the only reason (and you, of course). Thank you again.

**Comment by rpottsoh** [comment]
You're too kind, you are doing most of the work.  If you are interested you should consider mentoring.  http://mentoring.exercism.io

**Pull request merged**